### ISSUE TITLE
Data returned includes the Configuration data twice

### ISSUE THREAD

**Issue by kc1awv** [n7tae/dvin-tools#3]
Example:
```
{"Callsign":"M17-M17","Version":"0.10.0-dht","Modules":"ABCDEFGHIJKLMNOPQRSTUVWXYZ","EncryptMods":"ABCDEFGHIJKLMNOPQRSTUVWXYZ","IPv4Address":"152.70.192.70","IPv6Address":"2603:c020:4001:91ee::a1","URL":"https://ref.m17.link","Country":"US","Sponsor":"M17 Project","Email":"contact@m17.link","Port":17000}
{"Peers":[{"Callsign":"M17-202","Modules":"C","ConnectTime":"2022-12-09 12:00:09"},{"Callsign":"M17-

... truncated ... 

12-12 11:41:14","LastHeardTime":"2022-12-12 11:41:14"}]
{"Callsign":"M17-M17","Version":"0.10.0-dht","Modules":"ABCDEFGHIJKLMNOPQRSTUVWXYZ","EncryptMods":"ABCDEFGHIJKLMNOPQRSTUVWXYZ","IPv4Address":"152.70.192.70","IPv6Address":"2603:c020:4001:91ee::a1","URL":"https://ref.m17.link","Country":"US","Sponsor":"M17 Project","Email":"contact@m17.link","Port":17000}
```

This is also present after patching with PR #2

**Comment by kc1awv** [comment]
After patching with PR #2, the repeated information starts a second JSON object that isn't closed properly, breaking the data.

**Comment by n7tae** [comment]
Yes. See my comment in the #1 issue. There is some issues that I am working on with the OpenDHT team. It might be that I'm doing something wrong in mrefd, we'll see. In the meantime you probably need to shutdown M17-M17 for a few minutes and let the dht network clear out orphaned data. (It's been orphaned because the node that putSigned() it is no longer on the dht.) You can follow the progress by checking `./dht-get m17-m17` until nothing is returned. (It should only take a minute or two, maximum.) Then when you reinstall M17-M17, you should only see one version of each section.

**Comment by kc1awv** [comment]
I stopped and waited for the data to clear, then restarted M17-M17. Now it's giving me sections 'out of order' and some duplicated.

Output below:
```smiller@herzeleid:~/git/kc1awv/dvin-tools
$ ./dht-get M17-M17 -j -g
time: 1.29 seconds

smiller@herzeleid:~/git/kc1awv/dvin-tools
$ ./dht-get M17-M17 -j -g
{"Callsign":"M17-M17","Version":"0.10.0-dht","Modules":"ABCDEFGHIJKLMNOPQRSTUVWXYZ","EncryptMods":"E","IPv4Address":"152.70.192.70","IPv6Address":"2603:c020:4001:91ee::a1","URL":"https://ref.m17.link","Country":"US","Sponsor":"M17 Project","Email":"contact@m17.link","Port":17000,
"Peers":[],
"Clients":[]}
time: 1.44 seconds

smiller@herzeleid:~/git/kc1awv/dvin-tools
$ ./dht-get M17-M17 -j -g
{"Callsign":"M17-M17","Version":"0.10.0-dht","Modules":"ABCDEFGHIJKLMNOPQRSTUVWXYZ","EncryptMods":"E","IPv4Address":"152.70.192.70","IPv6Address":"2603:c020:4001:91ee::a1","URL":"https://ref.m17.link","Country":"US","Sponsor":"M17 Project","Email":"contact@m17.link","Port":17000,
"Peers":[],
"Clients":[]}
"Users":[{"Source":"KC1AWV  D","Destination":"KC1AWV  D","Reflector":"M17-M17 C","LastHeardTime":"2022-12-12 20:58:14"}],
"Peers":[{"Callsign":"M17-202","Modules":"C","ConnectTime":"2022-12-12 20:56:44"},{"Callsign":"M17-659","Modules":"C","ConnectTime":"2022-12-12 20:56:44"},{"Callsign":"M17-ITA","Modules":"A","ConnectTime":"2022-12-12 20:56:44"},{"Callsign":"M17-NOR","Modules":"A","ConnectTime":"2022-12-12 20:56:44"},{"Callsign":"M17-SPA","Modules":"A","ConnectTime":"2022-12-12 20:56:44"},{"Callsign":"M17-USA","Modules":"A","ConnectTime":"2022-12-12 20:56:49"}],
"Clients":[{"Module":"C","Callsign":"M17-659","IP":"2604:a880:800:14::fb:0","ConnectTime":"2022-12-12 20:56:44","LastHeardTime":"2022-12-12 20:56:44"},{"Module":"A","Callsign":"M17-NOR","IP":"185.35.185.100","ConnectTime":"2022-12-12 20:56:44","LastHeardTime":"2022-12-12 20:56:44"},{"Module":"C","Callsign":"M17-202","IP":"85.208.48.207","ConnectTime":"2022-12-12 20:56:44","LastHeardTime":"2022-12-12 20:56:44"},{"Module":"A","Callsign":"M17-SPA","IP":"81.2.248.38","ConnectTime":"2022-12-12 20:56:44","LastHeardTime":"2022-12-12 20:56:44"},{"Module":"A","Callsign":"M17-ITA","IP":"80.211.190.216","ConnectTime":"2022-12-12 20:56:44","LastHeardTime":"2022-12-12 20:56:44"},{"Module":"A","Callsign":"M17-USA","IP":"2600:1f16:d6f:600:20eb:5765:b94c:aebb","ConnectTime":"2022-12-12 20:56:49","LastHeardTime":"2022-12-12 20:56:49"},{"Module":"C","Callsign":"KC1AWV A","IP":"147.135.85.225","ConnectTime":"2022-12-12 20:56:53","LastHeardTime":"2022-12-12 20:56:53"},{"Module":"C","Callsign":"KE6IGZ C","IP":"45.19.167.101","ConnectTime":"2022-12-12 20:56:53","LastHeardTime":"2022-12-12 20:56:53"}]}
"Clients":[{"Module":"C","Callsign":"M17-659","IP":"2604:a880:800:14::fb:0","ConnectTime":"2022-12-12 20:56:44","LastHeardTime":"2022-12-12 20:56:44"},{"Module":"A","Callsign":"M17-NOR","IP":"185.35.185.100","ConnectTime":"2022-12-12 20:56:44","LastHeardTime":"2022-12-12 20:56:44"},{"Module":"C","Callsign":"M17-202","IP":"85.208.48.207","ConnectTime":"2022-12-12 20:56:44","LastHeardTime":"2022-12-12 20:56:44"},{"Module":"A","Callsign":"M17-SPA","IP":"81.2.248.38","ConnectTime":"2022-12-12 20:56:44","LastHeardTime":"2022-12-12 20:56:44"},{"Module":"A","Callsign":"M17-ITA","IP":"80.211.190.216","ConnectTime":"2022-12-12 20:56:44","LastHeardTime":"2022-12-12 20:56:44"},{"Module":"A","Callsign":"M17-USA","IP":"2600:1f16:d6f:600:20eb:5765:b94c:aebb","ConnectTime":"2022-12-12 20:56:49","LastHeardTime":"2022-12-12 20:56:49"},{"Module":"C","Callsign":"KC1AWV A","IP":"147.135.85.225","ConnectTime":"2022-12-12 20:56:53","LastHeardTime":"2022-12-12 20:56:53"},{"Module":"C","Callsign":"KE6IGZ C","IP":"45.19.167.101","ConnectTime":"2022-12-12 20:56:53","LastHeardTime":"2022-12-12 20:56:53"},{"Module":"C","Callsign":"KC1AWV  D","IP":"172.58.222.144","ConnectTime":"2022-12-12 20:58:08","LastHeardTime":"2022-12-12 20:58:08"}]}
"Peers":[],
"Clients":[]}
time: 1.57 seconds```

**Comment by n7tae** [comment]
When I finally got to the point of checking this just now, dht-get is returning a single value for each section for M17-M17. That makes me feel better!

I have worked out in my head how to deal with ordering the different sections.

I also have an idea how to deal with the case where multiple values of a single section are returned, but for that I need mrefd to include a timestamp or version number in each section when it does the putSigned() for each section. That way, dht-get will be able to tell which is the most up-to-date.
